lib/api/src/grpc/proto/points.proto

```diff
@@ -47,8 +47,9 @@ message SparseIndices {
 }
 
 message Vector {
-  repeated float data = 1;
-  optional SparseIndices indices = 2;
+  repeated float data = 1; // Vector data (flatten for multi-dimensional vectors)
+  optional SparseIndices indices = 2; // Sparse indices for sparse vectors
+  optional uint32 tokens_count = 3; // Number of tokens in the vector
```

Comment:
tokens is ColBERT specific terminology. It is possible that this multi-vector functionality will be used for other purposes then ColBERT (e.g. jsut storing embeddings of multiple images of the same object), so let's call it more generic